internal/util/streamingutil/service/balancer/balancer.go

```diff
@@ -0,0 +1,242 @@
+/*
+ *
+ * Copyright 2017 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Modified by github.com/milvus-io/milvus, @chyezh
```

Comment:
The code is modified from grpc v1.57.1.
Apache LICENSE 2.0

We need to achieve a unready connection sync balancer.
modification is described here.